app/helpers/application_helper.rb

```diff
@@ -5,14 +5,16 @@ def render_document_heading; end
 
   # Outputs the HTML to render a single value as an HTML table row
   # rubocop:disable Rails/OutputSafety
-  def render_field_row(title, value, render_empty: false)
+  def render_field_row(title, value, sidebar = false)
```

Comment:
Notice that this is a change to the meaning of the third parameter to this function, but, the parameter was not used and it was **incorrectly declared!!!!** so there is no harm in repurposing it.